packages/chrome/src/elements/subnav/CollapsibleSubNavItem.tsx

```diff
@@ -18,8 +18,15 @@ import {
 
 export interface ICollapsibleSubNavItemProps
   extends Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'onChange'> {
+  /** Sets the item header */
   header?: React.ReactNode;
+  /** Reveals the item text */
```

Comment:
Replace ~item text~ with something like "item's content" or "sub-items" (not sure that's a term we want to use 😝 ) or, similar to [Accordion](https://zendeskgarden.github.io/react-components/storybook/?path=/docs/components-accordions-accordion--uncontrolled) "item's section" (as this is technically an accordion control).

See the demo at https://garden.zendesk.com/components/chrome#navigation-panel with "show subnav" activated for additional working details for a `CollapsibleSubNavItem`.